src/assets/loader/parsers/loadWebFont.ts

```diff
@@ -20,6 +20,11 @@ const validFontMIMEs = [
     'font/woff2',
 ];
 
+export const FontCache = new Map<string, {
+    url: string,
+    fontFaces: FontFace[],
+}>();
+
```

Comment:
do we need to handle the case for when a font gets unloaded by assets? e.g. `Assets.unload('font.woof')`?